lms/djangoapps/certificates/models.py

```diff
@@ -998,6 +999,13 @@ class CertificateTemplate(TimeStampedModel):
         help_text=_(u'On/Off switch.'),
         default=False,
     )
+    language = models.CharField(
+        max_length=2,
+        choices=LANGUAGES,
```

Comment:
By including `choices` in the model, and incorporating them in the migration, are we restricting the ability of open-source users to customize the language-list? Does doing this also make it more difficult for us to add more languages later (we'd need a migration)?

Reply:
@bderusha , what would be the best place to incorporate a validation like this without it being part of the database?